src/hex/parser.ts

```diff
@@ -0,0 +1,33 @@
+import {DefaultSimpleParser} from '../parser';
+import {DicePool, dicePoolMonoid} from './dice';
+
+function letterToRolls(letter: string, occurrences: number): DicePool {
+    if (letter === 'h') {
+        return new DicePool(occurrences, 0, 0, 0, 0, 0, 0);
+    } else if (letter === '+') {
+        return new DicePool(0, occurrences <= 5 ? occurrences : 5, 0, 0, 0, 0, 0);
```

Comment:
I wouldn't inline ternary ifs, put them on a separate line since it's very hard to see where it starts and ends.

Reply:
Can you explain why you roll 5 if there are less or requal than 5 dice?

Reply:
Ok, I think there you read the inline if wrong:
occurrences <= 5 ? occurrences : 5
If there are less than (or equal) 5 occurances, I roll "occurances", otherwise I roll 5 (because both bonus and malus get capped at 5 _before_ they get netted.

Reply:
yeah that's the danger of ternary ifs :)